src/App.js

```diff
@@ -15,10 +16,14 @@ import TypeOfYe from "./pages/TypeOfYe";
 import KLKNotFound from "./pages/KLKNotFound";
 
 const App = () => {
+  const [userInput, setUserInput] = useState("");
+  const handleInput = (event) => {
+    setUserInput(event.target.value);
+  };
   return (
     <div className='App'>
+      <h1></h1>
```

Comment:
Do you need this empty h1 tag?

Reply:
> Do you need this empty h1 tag?

We just got rid of that. Thank you for pointing out.